client/src/components/ExploreCard.js

```diff
@@ -4,85 +4,82 @@ import { makeStyles } from "@material-ui/core/styles";
-	emptyTopSpace: {
-		paddingTop: "23rem",
-	},
-	cardDataContainer: {
-		borderTop: `1px solid ${theme.palette.primary.light}`,
-	},
-	locationText: {
-		color: "white",
-		fontSize: "23px",
-		margin: "1rem 0rem 0rem 2rem",
-		padding: "0rem",
-	},
-	locationCountryText: {
-		color: `${theme.palette.primary.light}`,
-		fontSize: "19px",
-		margin: "0rem 0rem 1.5rem 2rem",
-	},
+  mainPaper: (props) => ({
+    background: `linear-gradient(to bottom, rgba(0, 0, 0, 0) 50%, rgba(0, 0, 0, 0.6) 90%, rgba(0, 0, 0, 0.65) 100%), url(https://team-kim-possible.s3.us-east-2.amazonaws.com/images/${props.imgName}) center center/cover no-repeat `,
+    borderRadius: "18px",
+  }),
+  emptyTopSpace: {
+    paddingTop: "23rem",
+  },
+  cardDataContainer: {
+    borderTop: `1px solid ${theme.palette.primary.light}`,
+  },
+  locationText: {
+    color: "white",
+    fontSize: "23px",
+    margin: "1rem 0rem 0rem 2rem",
+    padding: "0rem",
+  },
+  locationCountryText: {
+    color: `${theme.palette.primary.light}`,
+    fontSize: "19px",
+    margin: "0rem 0rem 1.5rem 2rem",
+  },
 
-	favoriteIcon: {
-		marginRight: "2rem",
-		height: "2rem",
-		width: "2rem",
-		color: "white",
-	},
-	favoriteIconLiked: {
-		color: `${theme.palette.primary.main}`,
-		marginRight: "2rem",
-		height: "2rem",
-		width: "2rem",
-	},
+  favoriteIcon: {
+    marginRight: "2rem",
+    height: "2rem",
+    width: "2rem",
+    color: "white",
+  },
+  favoriteIconLiked: {
+    color: `${theme.palette.primary.main}`,
+    marginRight: "2rem",
+    height: "2rem",
+    width: "2rem",
+  },
 }));
 
 function ExploreCard(props) {
-	const classes = exploreCardStyles(props);
+  const classes = exploreCardStyles(props);
+  console.log(props);
 
-	const [liked, setLike] = useState(false);
+  const [liked, setLike] = useState(false);
 
-	const handleLiked = () => {
-		setLike((prev) => !prev);
-		// props.onLike();
-	};
-
-	return (
-		<Paper elevation={1} className={classes.mainPaper}>
-			<Grid container direction="column" justify="center">
-				<Grid item xs={12} className={classes.emptyTopSpace}></Grid>
-				<Grid container item xs={12} className={classes.cardDataContainer}>
-					<Grid item xs={6} container justify="flex-start" alignItems="center">
-						<Grid item xs={12}>
-							<p className={classes.locationText}>{props.location},</p>
-						</Grid>
-						<Grid item xs={12}>
-							<p className={classes.locationCountryText}>{props.country}</p>
-						</Grid>
-					</Grid>
-					<Grid item xs={6} container justify="flex-end" alignItems="center">
-						{liked ? (
-							<FavoriteIcon
-								className={classes.favoriteIconLiked}
-								onClick={() => handleLiked()}
-							></FavoriteIcon>
-						) : (
-							<FavoriteIcon
-								className={classes.favoriteIcon}
-								onClick={() => handleLiked()}
-							></FavoriteIcon>
-						)}
-					</Grid>
-				</Grid>
-			</Grid>
-		</Paper>
-	);
+  const handleLiked = () => {
+    setLike((prev) => !prev);
+    // props.onLike();
```

Comment:
I see we still haven't figured this out lol.
Also I'm much more of a fan of destructuring the specific functions/variables  instead of just passing props and doing `props.`, so if you do work with this, please make that change as well.